From 2713d825104627b7e273f90de51feab6b02e388b Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Fri, 28 Sep 2018 15:09:40 +0800
Subject: [PATCH] arm64: dts: rockchip: fixes the rk808's vddio-supply on
 rk3399-sapphire

From the schematic RK_SAPPHIRE_SOCBOARD_RK3399_LPDDR3D178P232SD8_V12,
the vcc33_3v0 supply power for vddio, and rk808's i2c used the 3.0v power.

Change-Id: Ib7546a100f6b6cf5406f4bd24a27db79bcdf4fc3
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-sapphire.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-sapphire.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-sapphire.dtsi
index 3d76b9733665..5728818893c2 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-sapphire.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-sapphire.dtsi
@@ -325,7 +325,7 @@
 		vcc10-supply = <&vcc3v3_sys>;
 		vcc11-supply = <&vcc3v3_sys>;
 		vcc12-supply = <&vcc3v3_sys>;
-		vddio-supply = <&vcc1v8_pmu>;
+		vddio-supply = <&vcc_3v0>;
 
 		regulators {
 			vdd_center: DCDC_REG1 {
-- 
2.35.3

